mwdb/resources/user.py

```diff
@@ -477,6 +478,57 @@ def put(self, login):
         schema = UserSuccessResponseSchema()
         return schema.dump({"login": user_login_obj["login"]})
 
+    @requires_authorization
+    @requires_capabilities(Capabilities.manage_users)
+    def delete(self, login):
+        """
+        ---
+        summary: Delete user
+        description: |
+            Remove user from database.
+
+            Requires `manage_users` capability.
+        security:
+            - bearerAuth: []
+        tags:
+            - user
+        parameters:
+            - in: path
+              name: login
+              schema:
+                type: string
+              description: User login
+        responses:
+            200:
+                description: When user was removed successfully
+                content:
+                  application/json:
+                    schema: UserSuccessResponseSchema
+            403:
+                description: When user doesn't have `manage_users` capability.
+            404:
+                description: When user doesn't exist.
+        """
+        if g.auth_user.login == login:
+            raise Forbidden("You can't remove yourself from the database.")
+        user = db.session.query(User).filter(User.login == login).first()
+        group = (
+            db.session.query(Group)
+            .options(joinedload(Group.members), joinedload(Group.members, Member.user))
```

Comment:
That joinedload seems to be not required because we don't operate on members :thinking:
```suggestion
```